docs/front-end/widget/README.md

```diff
@@ -0,0 +1,159 @@
+# CowSwap Widget
```

Comment:
```suggestion
# CoW Swap Widget
```

Reply:
Thank you!
Fixed everywhere: https://github.com/cowprotocol/docs-v2/pull/25/commits/96928b27ce5d6bf689ef7182af43e5d609b3fd05